config/initializers/resque_config.rb

```diff
@@ -1,5 +1,4 @@
-config = YAML::load(ERB.new(IO.read(File.join(Rails.root, 'config', 'redis.yml'))).result)[Rails.env].with_indifferent_access
-Resque.redis = Redis.new(host: config[:host], port: config[:port], thread_safe: true)
+Resque.redis = $redis
```

Comment:
Why go and grab the information again and setup yet another redis connection? You already setup redis in the redis initializer...